src/Templates/src/templates/maui-multiproject/MauiApp.1.WinUI/MauiApp.1.WinUI.csproj

```diff
@@ -16,6 +16,8 @@
 		<UseMaui>true</UseMaui>
 		<!-- We do not want XAML files to be processed as .NET MAUI XAML, but rather WinUI XAML. -->
 		<EnableDefaultMauiItems>false</EnableDefaultMauiItems>
+		<!-- To create a packaged Windows application delete the next line and update your launch settings.  See https://aka.ms/MauiTemplateUnpackaged. -->
```

Comment:
Can this comment be more explicit what to do? What should I change in `Properties/launchSettings.json`?